jina/hubble/hubio.py

```diff
@@ -50,7 +81,7 @@ def push(self) -> None:
 
         try:
             # archive the executor package
-            with TimeContext(f'archiving {self.args.path}', self.logger):
+            with TimeContext(f"archiving {self.args.path}", self.logger):
```

Comment:
```suggestion
            with TimeContext(f'archiving {self.args.path}', self.logger):
```

it looks like you are using Black but without `-S`, hence it converts all single quote to double quote